.travis.yml

```diff
@@ -45,3 +45,23 @@ matrix:
   - name: "Clang Release"
     compiler: clang
     config: Release
+  - &freebsd_common
+    name: "Clang 10 + system libraries"
+    os: freebsd
+    compiler: clang
+    before_install:
+      - | # FreeBSD 12.2 has Clang 10 but TravisCI uses FreeBSD 12.1
+        if [[ $(${CC:-clang} --version) =~ 8.0 ]]; then
+          export CC=clang10 CXX=clang++10
+          sudo pkg install -y llvm10
+        fi
+    before_script:
+      - sudo pkg install -y aom dav1d librav1e ninja
```

Comment:
Could we try using
```yml
      addons:
        pkg:
          packages:
            - aom
            - dav1d
            - librav1e
            - ninja
```
?

I haven't tested it out yet so I will quickly make sure that it's the right syntax
https://github.com/travis-ci/travis-yml/pull/154/files

Reply:
A few days ago it didn't work, except in recently created forks. Besides, `before_script` here overrides topmost definition (for Linux).

Reply:
I see, if pkg does work, we could just do `before_script: true` or `before_script: skip`

ps: seems to work

```diff
diff --git a/.travis.yml b/.travis.yml
index e44b654..d10e7b5 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -49,14 +49,20 @@ matrix:
     name: "Clang 10 + system libraries"
     os: freebsd
     compiler: clang
+    addons:
+      pkg:
+        packages:
+          - aom
+          - dav1d
+          - librav1e
+          - ninja
     before_install:
       - | # FreeBSD 12.2 has Clang 10 but TravisCI uses FreeBSD 12.1
         if [[ $(${CC:-clang} --version) =~ 8.0 ]]; then
           export CC=clang10 CXX=clang++10
           sudo pkg install -y llvm10
         fi
-    before_script:
-      - sudo pkg install -y aom dav1d librav1e ninja
+    before_script: skip
     script:
       - cmake -Bbuild -GNinja -DAVIF_CODEC_AOM=ON -DAVIF_CODEC_DAV1D=ON -DAVIF_CODEC_RAV1E=ON -DAVIF_BUILD_APPS=ON -DAVIF_BUILD_TESTS=ON
       - cmake --build build
```
![image](https://user-images.githubusercontent.com/8345542/82759901-c05a3480-9db5-11ea-8501-e6a968101d14.png)

Reply:
> ps: seems to work

Needs to be tested in a PR, not a fork. See https://github.com/Alexays/Waybar/commit/8d8c0489243fe2731161820e3013da7bf1ef434d.

Reply:
Hmm, interesting, wonder why it doesn't work. Maybe that should be reported upstream to travis?

Reply:
Found it. [This build](https://travis-ci.org/github/Alexays/Waybar/jobs/688252503) used `pkg` but failed, despite working in a fork.